sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OnHeapColumnVector.java

```diff
@@ -366,6 +396,12 @@ public double getDouble(int rowId) {
     }
   }
 
+  @Override
+  public void getDoubles(double[] dst, int rowId, int count) {
+    assert(dictionary == null);
+    System.arraycopy(byteData, rowId, dst, 0, count);
+  }
+
```

Comment:
Those lines seems to cover only `toIntArray` and `getInts`.
Is there any other test cases covering the other types like`toDoubleArray` and `getDoubles`?